app/controllers/SessionExpiredController.scala

```diff
@@ -32,12 +32,11 @@ class SessionExpiredController @Inject() (
 
   def onPageLoad(): Action[AnyContent] = Action {
     implicit request =>
-      Ok(view()).withNewSession
+      Ok(view())
   }
 
   def onSubmit(): Action[AnyContent] = Action {
     _ =>
-      val signInUrl = s"${config.manageTransitMovementsUrl}/what-do-you-want-to-do"
-      Redirect(signInUrl)
+      Redirect(s"${config.manageTransitMovementsUrl}/what-do-you-want-to-do")
```

Comment:
@jackheslop96 @carrielies to ensure the user is signed out we need to retain the `.withNewSession` method call.

Reply:
I've updated that to add the withNewSession when the page submits. It was previously doing it onLoad which caused some intermittent failures when manually testing it